app/workers/streams/twitch_webhook_registration_worker.rb

```diff
@@ -1,6 +1,7 @@
 module Streams
-  class TwitchWebhookRegistrationJob < ApplicationJob
-    queue_as :twitch_webhook_registration
+  class TwitchWebhookRegistrationWorker
+    include Sidekiq::Worker
+    sidekiq_options queue: :default, retry: 10
 
     def perform(user_id, service = TwitchWebhook::Register)
       user = User.find_by(id: user_id)
```

Comment:
In #5305 you mentioned having less jobs fail silently. We shouldn't be enqueuing jobs for non-existent users too often, so I was wondering if we should slowly move to `find` for situations like this. As a less harsh approach, maybe we could at least report the `nil` somewhere, so we have an insight into how often this happens and whether or not it changes.

Reply:
I think the plan here is to move the jobs over to Sidekiq first and then once we have them all on the new queue and then go over all of them and decide which ones should fail silently and which one shouldn't

Reply:
Seems like a perfectly sensible idea, which is why I suggested potentially tracking these in order to simplify the decision afterwards.